jiant/tasks/senteval.py

```diff
@@ -0,0 +1,530 @@
+""" 
+Set of tasks from Senteval
```

Comment:
Searching through that paper, it never describes these probing tasks as being part of SentEval.  Probably okay to use something like `senteval-probing` as a short name/filename, but we shouldn't use just `senteval`, and we should add a note here to clarify that we mean these probing tasks that were later added to the senteval repo.